README.md

```diff
@@ -24,3 +24,4 @@ Join the conversation on **[Gitter](https://gitter.im/Microsoft/BotBuilder)**.
 See all the support options **[here](https://docs.microsoft.com/en-us/bot-framework/resources-support)**.
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+# gdpr
```

Comment:
Do you want more than this?  It would show up on the main page of the GitHub repository.